### PR TITLE
Feature/endorse-config with steps

### PR DESCRIPTION
**Pull request by Jsyro**
Docker-compose, config, and readme to demonstrate native steps to successfully set up endorser/author connection/relationship. 

These steps will need to built into the controllers in the future.

### CODE REVIEW

File: scripts/endorser-connection.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+   "transaction_jobs": {
+      "transaction_their_job": "TRANSACTION_ENDORSER",
+      "transaction_my_job": "TRANSACTION_AUTHOR"
+    }
+	}
+}
+
+```
+
+3. Set endorsers connection to author/tenant with the following. POST to `/connections/{conn_id}/metadata`
```
Comment (ianco) [inline scripts/endorser-connection.md:22]: This is for the Endorser (not author/tenant).  Use the endpoint `/transactions/{conn_id}/set-endorser-role`

File: scripts/endorser-connection.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+To use the endorser agent in this project the following steps need to happen. Through swagger pages. 
+
+1. Generate invitation from endorser (POST `/connections/create_invitation`, receive invitation from tenant `/connections/recieve-invitation`
```
Comment (ianco) [inline scripts/endorser-connection.md:3]: You can create the invitation in advance and then use the `--endorser-invitation` aca-py parameter, but not sure how to set this up in docker-compose.  Possibly something to consider for the OpenShift deployment (where we can deploy/configure the Endorser agent in advance)

File: scripts/endorser-connection.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+{
+  "metadata":{
+    "transaction_jobs": {
+      "transaction_my_job": "TRANSACTION_ENDORSER",
+      "transaction_their_job": "TRANSACTION_AUTHOR"
+    }
+  }
+}
+```
+4. Out Of Band: Get tenant did/verkey from (with bearer token) GET `wallet/did` and POST that did/verkey to Endorser swagger at `/ledger/register-nym`.
```
Comment (ianco) [inline scripts/endorser-connection.md:33]: Note - I'm working on an update to aca-py to add Endorser support for this (publishing author DID).

**Review by ianco — CHANGES_REQUESTED** [review]

File: scripts/endorser-connection.md (hunk 2 above)
Comment (Jsyro) [reply scripts/endorser-connection.md:3]: will this `--endorser-invitation` parameter apply to the tenants in a multi-tenant configuration? if the a new tenant is created, will it automatically use this invitation, or will it need to prompted to use this 'default' invitation?

File: scripts/endorser-connection.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+To use the endorser agent in this project the following steps need to happen. Through swagger pages. 
+
+1. Generate invitation from endorser (POST `/connections/create_invitation`, receive invitation from tenant `/connections/recieve-invitation`
+
+1. Set author/tenant's connection to endorser with the following. POST to `/connections/{conn_id}/metadata`
```
[the remark this answers is not in the document]
Comment (Jsyro) [reply scripts/endorser-connection.md]: that makes more sense, updated

File: scripts/endorser-connection.md (hunk 1 above)
Comment (Jsyro) [reply scripts/endorser-connection.md]: Updated

File: scripts/endorser-connection.md (hunk 2 above)
Comment (ianco) [reply scripts/endorser-connection.md:3]: I'll look into this, if the tenant doesn't auto-connect to the endorser I'll fix it ...

**Pull request merged**